pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>com.amazonaws</groupId>
   <artifactId>amazon-sqs-java-extended-client-lib</artifactId>
-  <version>2.0.4</version>
+  <version>2.0.5</version>
```

Comment:
Please also update the `1.x` branch as that will be what the customer is asking for.